src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.csproj

```diff
@@ -1,14 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Configurations>netcoreapp-Debug;netcoreapp-Release</Configurations>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Text.RegularExpressions.cs" />
   </ItemGroup>
   <!-- Files required to enable compiled feature -->
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
```

Comment:
This seems wrong. Should probably revert?

Reply:
Actually, per my previous comment, this looks like the right thing to do (along with deleting the .netcoreapp.cs file.  The only configuration we now have is netcoreapp, so splitting the files no longer makes sense.